src/pages/Profile.tsx

```diff
@@ -144,23 +145,24 @@ const Profile = () => {
     setProfileContents((prevProfileContents) => ({
       ...prevProfileContents,
       followingCount: isFollowing
-        ? --prevProfileContents.followingCount
-        : ++prevProfileContents.followingCount,
+        ? prevProfileContents.followingCount - 1
+        : prevProfileContents.followingCount + 1,
```

Comment:
I don't think this actually fixes the bug. If you unfollow someone, refresh the page, then follow them again from the modal it appends to the string (ie. "4" turns to "41" instead of "5"). I debugged it and the reason why we're having so much trouble with this stuff is because the back end returns a string instead of a number, so we're trying to increment/decrement a string

I don't have an answer for question of whether the back end should return numbers as strings in the first place, but the temporary fix for this is to cast it to a number first

```suggestion
        ? Number(prevProfileContents.followingCount) - 1
        : Number(prevProfileContents.followingCount) + 1,
```